File: .automaker/deploy/hello-world.js
#!/usr/bin/env node

/**
 * Hello World Deploy Script (Node.js)
 *
 * This is a simple example deploy script that demonstrates:
 * - Basic script structure
 * - Environment variable access
 * - Console output that gets captured by the deploy service
 */

console.log('='.repeat(60));
console.log('🚀 Hello World Deploy Script');
console.log('='.repeat(60));
console.log('');

console.log('📦 Project Directory:', process.cwd());
console.log('🕒 Timestamp:', new Date().toISOString());
console.log('🖥️  Platform:', process.platform);
console.log('📍 Node Version:', process.version);
console.log('');

console.log('✅ Deploy script executed successfully!');
console.log('');
console.log('This is where you would typically:');
console.log('  - Build your application');
console.log('  - Run tests');
console.log('  - Upload artifacts');
console.log('  - Deploy to servers');
console.log('  - Send notifications');
console.log('');

console.log('='.repeat(60));
console.log('✨ Deployment Complete!');
console.log('='.repeat(60));

// Exit with success code
process.exit(0);

File: .automaker/deploy/hello-world.py
#!/usr/bin/env python3

"""
Hello World Deploy Script (Python)

This is a simple example deploy script that demonstrates:
- Basic script structure
- Environment variable access
- Console output that gets captured by the deploy service
"""

import os
import sys
from datetime import datetime
import platform

def main():
    border = "=" * 60

    print(border)
    print("🚀 Hello World Deploy Script (Python)")
    print(border)
    print()

    print(f"📦 Project Directory: {os.getcwd()}")
    print(f"🕒 Timestamp: {datetime.now().isoformat()}")
    print(f"🖥️  Platform: {platform.system()} {platform.release()}")
    print(f"📍 Python Version: {sys.version.split()[0]}")
    print()

    print("✅ Deploy script executed successfully!")
    print()
    print("This is where you would typically:")
    print("  - Build your application")
    print("  - Run tests with pytest")
    print("  - Upload artifacts to S3/GCS")
    print("  - Deploy to cloud platforms")
    print("  - Send Slack/Discord notifications")
    print()

    print(border)
    print("✨ Deployment Complete!")
    print(border)

    # Exit with success code
    return 0

if __name__ == "__main__":
    sys.exit(main())
